pkg/security/ImageScanService.go

```diff
@@ -105,11 +147,21 @@ func (impl *ImageScanServiceImpl) ScanImage(scanEvent *common.ImageScanEvent, to
 		impl.logger.Infow("image already scanned, skipping further process", "image", scanEvent.Image)
 		return nil
 	}
-	imageScanRenderDto, err := impl.getImageScanRenderDto(scanEvent.DockerRegistryId, scanEvent.Image)
+	var caCertFilePath string
+	if scanEvent.DockerConnection == common.SECUREWITHCERT {
```

Comment:
how we are handling insecure case ?

Reply:
for insecure case, we are adding a key `insecure` in jsonMap that we are using in the cli command (command in sql migration script)